### PR TITLE
New Role Suggestions - Colour Faction Roles

### PR DESCRIPTION
**Pull request by TheEighthDeadlySin**
Apparently renaming the branch deleted the pull request... At the moment, the only thing I have changed due to Luffy's thoughts is the Jester's respawn time (increased from 2 to 4). Sorry about the earlier pull request disappearing...

### CODE REVIEW

**Comment by TheEighthDeadlySin** [comment]
I am not sure whether Jester should also have any attacking items, such as Bombs, that are planted on their target redirected to them. So for now, I have only included that Jester will only redirect abilities, and items used on their target will not be redirected (so Bombs, Poison, Antidote etc. will bypass this ability).

**Review by Topkinsme — CHANGES_REQUESTED** [review]
Warlord sounds good, but has the same problems Luffy earlier mentioned, with it just being killer but different. Also, do not have a "Winning Point" statement, just make it another point

Jester sounds good, but it should not be phase end, it should be that they pick a target during the day and it works for the entire next night.

Mage sounds good, But I recommend changing all instances of Basic Prot to weak prot, as it might be confusing if we add an actual basic prot status

Siege_warrior sounds good, however I feel like it should be a day action, as in the siege is planned during the day and executed during the night instead of it being instant.

Princess sounds good

Maybe we could drop sacrificial lamb's respawn, as it's only a 1-time thing.

**Comment by TheEighthDeadlySin** [comment]
Okay, I shall implement the changes right now!

**Comment by TheEighthDeadlySin** [comment]
Changes implemented:
Jester: Action is now performed during the day, any item or ability used throughout the night (includes immediate actions) on their target will instead be redirected to the Jester.
Mage: Renamed Basic Protection to Weak Protection.
Siege Warrior: Changed to action is performed during the day, and its effects are applied the following night.
Sacrificial Lamb: Decreased Respawn Time from 6 to 4.

**Review by Topkinsme — CHANGES_REQUESTED** [review]
Jester shouldn't redirect items, I only meant it should last the entire night instead of being an instant action

Everything else is good

**Comment by TheEighthDeadlySin** [comment]
Okay, I will change Jester so only abilities (both immediate and end phase) are redirected. Items will not be affected.

**Comment by TheEighthDeadlySin** [comment]
Removed Warlord

**Comment by TheEighthDeadlySin** [comment]
I have added two Solo Chaos Role Suggestions: Gambler and Bounty Hunter. I have also checked to make sure all descriptions have been written in the third person.

**Comment by Topkinsme** [comment]
I'd recommend you make a new pull req with all the solo roles like Warlord, Bounty Hunter, and gambler, since we're close to approving the other roles and we could implement the other roles and not worry about the solos for now

**Comment by TheEighthDeadlySin** [comment]
Okay, I shall create a new role suggestion pull request containing the three solo chaos roles.

**Comment by TheEighthDeadlySin** [comment]
Removed non-approved Solo Suggestions to a new pull request: https://github.com/Topkinsme/Color-Battle-Roles/pull/33

File: strong_power/jester [hunk 1]
```diff
@@ -0,0 +1,10 @@
+0. Jester
+- During the day, may choose one player. The following night, all actions performed on the player will be redirected onto themself.
+- They cannot change their target during the night.
+  Also may not choose to redirect on the same player for two nights in a row.
+  Action has no cooldown.
+  Redirection ability will occur throughout the night. 
+  Only abilities (noth immediate and end phase attacks) on their target will be redirected to the Jester.
```
Comment (Topkinsme) [inline strong_power/jester:7]: "not" not "noth" typo

**Review by Topkinsme — APPROVED** [review]
Looks good to me, except that 1 typo

**Comment by Topkinsme** [comment]
Also, you can change Sacrificial Limb respawn to 2 phases if you want, as Luffy suggested.

File: strong_power/jester (hunk 1 above)
Comment (TheEighthDeadlySin) [reply strong_power/jester:7]: oops XD

File: weak_power/sacrificial_lamb [hunk 2]
```diff
@@ -2,4 +2,4 @@
 - When the Sacrificial Lamb is killed by tribute, every other player that was tributed by the other colour factions will also die.
 - The Passive Ability will only activate once.
 - No Actions, only a Passive Ability (explained above).
-- Respawns in 6 phases.
+- Respawns in 4 phases.
```
[the remark this answers is not in the document]
Comment (TheEighthDeadlySin) [reply weak_power/sacrificial_lamb:5]: Okay.

File: solo_chaos/bounty_hunter [hunk 3]
```diff
@@ -1,12 +1,12 @@
...
 - Every Night Phase will receive a choice of three players. 
   May only choose to kill one of them.
-  Every time you successfully kill a target, you will receive some money.
+  Every time they successfully kill a target, they will receive some money.
   Wins if they have received a net profit of $2000 through bounty killing.
-- Every successful kill will give you $200 of a bounty reward.
-  If you kill a King, you will instead receive $500 of a bounty reward.
-  Every attack will be a Weak Attack. [Until the Player purchases the item that changes from Weak to Strong Attacks.]
-  May buy two different items, which will deduct from the net profit of bounty rewards [may not purchase if net profit below price]:
+- Every successful kill will give them $200 of a bounty reward.
```
[the remark this answers is not in the document]
Comment (TheEighthDeadlySin) [reply solo_chaos/bounty_hunter]: The amount earned has to be from bounty killing, it does not count money earned from other sources, such as forge.

File: weak_power/sacrificial_lamb (hunk 2 above)
[the remark this answers is not in the document]
Comment (TheEighthDeadlySin) [reply weak_power/sacrificial_lamb:5]: I will decrease the respawn time to 2 phases.

File: strong_power/jester [hunk 4]
```diff
@@ -3,6 +3,8 @@
 - They cannot change their target during the night.
   Also may not choose to redirect on the same player for two nights in a row.
   Action has no cooldown.
-  Redirection ability will occur throughout the night. Any items and abilities (including instant attacks) on their target will be redirected to the Jester.
+  Redirection ability will occur throughout the night. 
+  Only abilities (noth immediate and end phase attacks) on their target will be redirected to the Jester.
+  Items used on their target will not be affected.
```
[the remark this answers is not in the document]
Comment (Topkinsme) [reply strong_power/jester:8]: Items don't work, so I don't think this applies

File: weak_power/sacrificial_lamb (hunk 2 above)
[the remark this answers is not in the document]
Comment (Topkinsme) [reply weak_power/sacrificial_lamb:5]: Agrred

File: strong_power/jester (hunk 4 above)
[the remark this answers is not in the document]
Comment (TheEighthDeadlySin) [reply strong_power/jester]: I put it in as a clarification (in case people thought Jester will also alter item usage).

**Review by Topkinsme — APPROVED** [review]
All good

**Comment by Topkinsme** [comment]
I'm approving this pull request, this looks good to me.

File: weak_power/sacrificial_lamb (hunk 2 above)
[the remark this answers is not in the document]
Comment (TheEighthDeadlySin) [reply weak_power/sacrificial_lamb]: Done!

**Pull request merged**